### PR TITLE
add webhook verify and parse endpoints

### PR DESCRIPTION
**Pull request by andymckay**
fixes mozilla/solitude#452 with the upcoming solitude patch

### CODE REVIEW

File: auth/views.py [hunk 1]
```diff
@@ -31,5 +36,47 @@ def braintree(request):
...
+    :bt_payload string: the payload from braintree.
+    :bt_signature string: the signature that needs to be verified.
+
+    :status 204: the data is to be trusted. Contains no content.
+
+    Any other status is a failure to verify and should not be trusted.
+    """
+    data = (
+        request.POST.get('bt_signature', ''),
+        request.POST.get('bt_payload', '')
```
Comment (kumar303) [inline auth/views.py]: since signature checking is usually binary do either of these need to be converted to ascii?

Comment (andymckay) [inline auth/views.py]: its all base64 encoded strings, no need

Comment (kumar303) [inline auth/views.py]: Well they will be unicode because Django. If the library doesn't complain then I'm happy :)

**Comment by kumar303** [comment]
r+wc

**Pull request merged**